lib/matplotlib/patches.py

```diff
@@ -4253,8 +4261,10 @@ def get_path_in_displaycoord(self):
         dpi_cor = self.get_dpi_cor()
 
         if self._posA_posB is not None:
```

Comment:
This seems good - is it worth looking at the other patches to see if we need to add this?  You could test with the same test as below and see if they fail?

Reply:
I don't have the bandwidth to do this at the moment; perhaps quite a nice thing to add as a new-contributor friendly issue though.